src/Raven.Server/Commercial/LicenseType.cs

```diff
@@ -8,12 +8,14 @@ public enum LicenseType
 
         Community = 1,
 
-        Trial = 2,
+        Reserved = 2,
 
         Professional = 3,
 
         Enterprise = 4,
 
-        Developer = 5
+        Developer = 5,
+
+        Essential = 6
```

Comment:
how would existing code handle this new license type?
IIRC, we aren't actually doing anything with it, right?

Reply:
Anything that cannot be recognized will be `Community`.
We use the `Developer` type to check the certificate expiration.
The other usage is displaying it in the studio.